src/js/files/components/__tests__/Manager.test.jsx

```diff
@@ -118,7 +120,7 @@ describe("<FileManager>", () => {
                     preloadedState: state,
                 }),
         );
-        const invalidFile = new File(["test"], "test_invalid_file.gz", { type: "application/gzip" });
+        const invalidFile = new File(["test"], "test_invalid_file.gzz", { type: "application/gzip" });
```

Comment:
Was the original not working? I think the expectation was that .gz would fail since the file type needs to have `.fa` or `.fasta` before the compression extension

Reply:
No you are right, this was left over from one of my previous attempts of fixing a failure